plugins/inputs/postgresql_extensible/postgresql_extensible.go

```diff
@@ -129,8 +146,18 @@ func (p *Postgresql) Gather(acc telegraf.Accumulator) error {
 	// We loop in order to process each query
 	// Query is not run if Database version does not match the query version.
 	for i := range p.Query {
+		if p.Query[i].Sqlquery != "" && p.Query[i].Script != "" {
+			return fmt.Errorf("both 'sqlquery' and 'script' specified, please choose one option")
```

Comment:
I would remove this and just document that `sqlquery` takes preference if both are defined.

Reply:
I don't have a strong opinion on returning an error vs precedence, though if I was asked to choose I think an error is slightly less prone to mistakes.  However, I would like the script loading to be done only once.  The easiest way to do this is to move it into a new Init() function:
```
func (p *Postgresql) Init() error {
    // load the file and save onto the Postgresql struct.
    // return an error if the file is unreadable or configuration is invalid
}
```